tests_integration/test_procedure.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+import pytest
+
+from tests_integration.snowflake_connector import snowflake_session, test_database
+from tests_integration.testing_utils.naming_utils import object_name_provider
+from tests_integration.testing_utils.snowpark_utils import (
+    SnowparkTestSteps,
+    SnowparkTestSetup,
+    TestType,
+)
+from tests_integration.testing_utils.sql_utils import sql_test_helper
+from tests_integration.testing_utils.working_directory_utils import (
+    temporary_working_directory,
+)
+
+
+@pytest.mark.integration
+def test_snowpark_procedure_flow(_test_steps):
+    _test_steps.assert_that_no_entities_are_in_snowflake()
+    _test_steps.assert_that_no_files_are_staged_in_snowflake()
```

Comment:
Staged anywhere or on particular stage?

Reply:
I changed "snowflake" to "test_db" - hope this will be clearer